hooks/pre-checkout

```diff
@@ -32,7 +32,7 @@ git_mirror_download () {
   #
   # To find the path for the current repo, we `grep` for its name at the start
   # of the line, using `^`.
-  if URL=$(curl "$GIT_MIRROR_SERVER_ROOT/manifest" | grep ^"$REPO_PATH"); then
+  if URL=$(curl "$GIT_MIRROR_SERVER_ROOT/manifest" | grep ^"$REPO_PATH" | head -n1); then
```

Comment:
Should we `| sort -r` the result at some point while at it, to remove an additional assumption (that file list in the manifest are always sorted with most recent cache first)?

Reply:
Added in https://github.com/Automattic/git-s3-cache-buildkite-plugin/pull/14/commits/23b734a2b8a64d8effd9e837063674835e9a4489